trame_vtk/modules/vtk/protocols/file_browser.py

```diff
@@ -0,0 +1,80 @@
+import os
```

Comment:
Are we using that protocol?

Reply:
Doesn't look like it. I just just transferred them all over. Do you want me to delete the unused ones?

Reply:
yes, so we only have the piece of code we really need to worry about

Reply:
Done.